### DISCUSSION TITLE
GitHub pages set "Content-Type: application/pgp-signature" for SSH signatures

### DISCUSSION THREAD

**Discussion by Mikaela** [community/community#30084] [Other Feature Feedback, Questions, & Ideas]
I am hosting a couple of SSH-signed text files on GitHub Pages and they seem to receive a `Content-Type: application/pgp-signature` while they are SSH signatures. I am not sure what Content-Type they should receive though.

For example `HEAD https://www.aminda.eu/txt/matrix.txt.sig|grep Content-Type` returns `Content-Type: application/pgp-signature`.

See also:

* [Andrew Ayer on November 12, 2021: It's Now Possible To Sign Arbitrary Data With Your SSH Keys](https://www.agwa.name/blog/post/ssh_signatures)
* [Allow using SSH keys to sign commits](https://github.com/orgs/community/discussions/7744)
* https://github.com/mozilla-mobile/fenix/issues/26543

**Comment by yoannchaudet** [comment]
Pages relies on [`mime-db`](https://github.com/jshttp/mime-db) for content types. So in your case the `Content-Type` you receive is coming from [here](https://github.com/jshttp/mime-db/blob/c9242a9b7d4bb25d7a0c9244adec74aeef08d8a1/db.json#L1192-L1195).

I don't know either what content type would work for that kind of file. Maybe just compress them?
